templates/service-template.yml

```diff
@@ -505,11 +505,6 @@ parameters:
   description: ID of the kas-fleetshard addon
   value: "kas-fleetshard-operator-qe"
 
-- name: CLUSTER_LOGGING_OPERATOR_ADDON_ID
```

Comment:
let's also remove https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/blob/1d61b2c5b14c046861aadb7be41db2aaf1c38be9/templates/service-template.yml#L1103

Reply:
Same as removing an entry for `CLUSTER_LOGGING_OPERATOR_ADDON_ID` from https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/blob/1d61b2c5b14c046861aadb7be41db2aaf1c38be9/docs/deploying-kas-fleet-manager-to-openshift.md#deploy-kas-fleet-manager